### PR TITLE
feat: add users model controller and refactor controller and model folders

### PR DESCRIPTION
**Pull request by prestonb91**

### CODE REVIEW

File: src/models/users.model.js [hunk 1]
```diff
@@ -5,9 +5,12 @@ const knex = require("../knex.js");
 class User {
     constructor() {}
 
-    static getAll() {
-        return knex.select();
+    static addUser(userData) {
+        return knex
+            .insert(userData)
+            .into("users")
```
Comment (brianaqp) [inline src/models/users.model.js:11]: "Users"

Comment (brianaqp) [inline src/models/users.model.js:11]: DB Tables are capital

**Review by brianaqp — CHANGES_REQUESTED** [review]
Perfect, just rename this.

**Comment by prestonb91** [comment]
@brianaqp updated the DB table name to be capital.

**Review by brianaqp — APPROVED** [review]

**Pull request merged**